functions/11.js

```diff
@@ -9,4 +9,19 @@
 export function adjustedSalary(salary, taxRate) {
   // TODO: apply the tax deduction
   // TODO: return the adjusted salary
+  return salary - (taxRate / 100) // this isn't complete- 1 pass and 4 failures 
```

Comment:
Try

```js
salary * (1 - taxRate / 100)
```

This works because, for example, reducing by 10% is the same as multiplying by 0.9.

Reply:
thanks!